pvlib/test/test_pvsystem.py

```diff
@@ -1118,6 +1118,23 @@ def test_pvsyst_celltemp_with_index():
     assert_series_equal(expected, pvtemps)
 
 
+def test_PVSystem_pvsyst_celltemp(mocker):
+    racking_model = 'insulated'
+    system = pvsystem.PVSystem(racking_model=racking_model)
+    mocker.spy(pvsystem, 'pvsyst_celltemp')
+    irrads = 1000
+    temps = 25
+    alpha_absorption = 0.9
```

Comment:
Use non-default parameters to ensure that the right values are passed to the function.